### DISCUSSION TITLE
there is no support for graphics tablet and mouse in the android version of the game

### DISCUSSION THREAD

**Discussion by FREINYES** [ppy/osu#18034] [Q&A] [answered]
I connected the mouse to the phone via OTG, it works on Android water and there are no settings for it in the game, I connected the graphics tablet via OTG to work in drawing programs and the game does not see it.

where to rearrange logs of game ??

**Comment by smoogipoo** [comment, marked as answer]
OpenTabletDriver doesn't support Android. We'll likely need platform-specific handling.

**Reply by frenzibyte** [reply]
The tablet part of this is actually tracked in https://github.com/ppy/osu/issues/17382, just realized it. In addition, the mouse part will be resolved in https://github.com/ppy/osu/pull/16464.

**Reply by FREINYES** [reply]
I hope that the settings of the game android version of the game will be the same as on the PC.
It is sad that the game will not support all graphics tablets
at least for the mouse blows its input

**Reply by bdach** [reply]
Please understand that using external tablets on mobile is a rare hardware configuration, requires special hardware for us to develop/test, and the mobile version is short on effort as is anyways. I wouldn't hold my breath that this is going to be supported anytime soon.

**Reply by FREINYES** [reply]
I understand everything perfectly
how to end the discussion and where and how to transfer the game logs?

**Reply by FREINYES** [reply]
I'm on github for the first time
so I don't know anything and I don't know English well

**Reply by bdach** [reply]
We don't really need logs in this particular case.

**Reply by FREINYES** [reply]
Ok
